src/fileformats/ocd_georef_fields.h

```diff
@@ -52,30 +50,26 @@ struct OcdStringFields
 	 * Equality operator comparing this structure with another. Field `a` gets
 	 * compared with 8-digit precision.
 	 */
-	bool operator==(const OcdStringFields& rhs) const;
-};
-
-/**
- * Fill in provided georef with data extracted from type 1039 string.
- * @param georef Reference to a Georeferencing object.
- * @param param_struct Data extracted from OCD type 1039 string.
- * @param warning_handler Function to handle import warnings.
- */
-void setupGeorefFromFields(Georeferencing& georef,
-                           const OcdStringFields& param_struct,
-                           const std::function<void (const QString&)>& warning_handler);
+	bool operator==(const OcdGeorefFields& rhs) const;
```

Comment:
Please make `operator==` a non-member function. Cf. [CppCoreGuidelines C.68](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c86-make--symmetric-with-respect-to-operand-types-and-noexcept). 
While at it, just add `operator!=` (in terms `operator==`).

This the final remark. Thanks again.

Reply:
Done.